ext/phar/stream.c

```diff
@@ -44,7 +44,11 @@ php_stream_wrapper_ops phar_stream_wops = {
 	phar_wrapper_unlink,   /* unlink */
 	phar_wrapper_rename,   /* rename */
 	phar_wrapper_mkdir,    /* create directory */
-	phar_wrapper_rmdir,    /* remove directory */
+	phar_wrapper_rmdir     /* remove directory */
+#ifdef PHP_STREAMS_SUPPORT_IS_CACHEABLE
```

Comment:
I did it this way because I imagined that the phar code had to be kept compatible with previous PHP versions (but maybe the phar reference is not here, or development for PHP7 has been split).

If we consider that this code is synchronized with is_cacheable() support, yes, we can drop the conditionals.